Makefile

```diff
@@ -9,6 +9,8 @@ DESTDIR=/
 
 all: build
 
+all-style:: fix reformat codespell
```

Comment:
style intentionally only reports on style, it doesn't make any changes. So it seems that all-style would.

fix makes actual code changes, it's not just a style thing.

Reply:
I went ahead and added separate `all-style` and `all-fix`. I also reordered the targets within `Makefile` so they're grouped by area; this doesn't change any of the rules, just makes them easier to scan to find the one you're interested in IMHO.